packages/api/src/typeDefs/product.graphql

```diff
@@ -70,6 +70,10 @@ type StoreProduct {
   Sku Unit Multiplier
   """
   unitMultiplier: Float
+  """
+  Advertisement information about the product.
+  """
+  advertisement: Advertisement
```

Comment:
In FastStore, we follow schema.org to define new props, and [schema.org does not have an advertisement prop for Product](https://schema.org/Product) that fits better for advertisement (maybe [additionalProperty](https://schema.org/additionalProperty) ?). In this case, I think we can proceed with the advertisement prop and propose this new field to schema.org somehow. What do you think?

Reply:
I didn't know we used a well-defined partner for our schemas, seems awesome.

I agree and I will study schema.org to see how `advertisement` will fit, but first I think we should test and see how it goes.